core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.css

```diff
@@ -147,8 +147,14 @@
 }
 .language-dropdown {
   max-height: 60vh;
+  max-width: 500px;
+  min-width: 90px;
   overflow-y: auto;
 }
+.language-dropdown.dropdown-menu.oppia-navbar-dropdown {
+  left: 5px !important;
```

Comment:
Why do we need !important?

Reply:
Thanks for reviewing,
I am sorry added it by mistake. Will remove it.
Thanks for correcting.